R/input-numeric.R

```diff
@@ -24,6 +24,10 @@
 #' }
 #' shinyApp(ui, server)
 #' }
+#'
+#' @section Server value:
+#' The input value as a `double`.
```

Comment:
```suggestion
#' A [numeric] vector of length 1.
```


It's possible for the input value to be an integer

```r
library(shiny)
ui <- fluidPage(numericInput("x", "Hey", 3L))
server <- function(input, output, session) {
  observe(print(str(input$x)))
}
shinyApp(ui, server)
```

```r
Listening on http://127.0.0.1:4368
 int 3
```

Reply:
Actually, I guess it's more correct to say `A length 1 vector with a [mode] of `"numeric"``

Reply:
I don't think it's a good idea to talk about the "mode" of a vector since "mode" is mostly an S concept. I think a numeric vector of length 1 is fine (although I wouldn't link to `numeric()` since that's an alias for `double()` which isn't the meaning you want to imply here)